Gemfile

```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
```

Comment:
you missed gem installation so the Gemfile.lock is missing. Please do it and update PR again.